index.md

```diff
@@ -15,8 +15,8 @@ customers happy.
     <p>Start designing with Helix components.</p>
     <div class="card-bottom">
       <ul>
-        <li><a href="{{site.url}}">Pattern Guidelines</a></li>
-        <li><a href="https://github.com/rackerlabs/helix/">Sketch Resources</a></li>
+        <li><a href="{{site.url}}/status.html">Pattern Guidelines</a></li>
+        <li><a href="{{site.cdn_url}}/sketch/helix_stickersheet_v0.1.sketch">Sketch Resources <hx-icon type="download"></hx-icon></a></li>
```

Comment:
do we really want to version the sticker sheet via the link?
might be better to make this a permanent un-versioned link, version within the target

Reply:
At the moment, that's what it's titled as. I think there was a discussion about that during one of the team meetings; personally, I prefer to check into version control and drop the version in the title, but that's me :)